.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,105 @@
+name: publish distributions
+on:
+  push:
+    branches:
+    - main
+    tags:
+    - v*
+  pull_request:
+    branches:
+    - main
+    - release/v*
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      publish:
+        type: choice
+        description: 'Publish to TestPyPI?'
+        options:
+        - false
+        - true
```

Comment:
This would allow you to create a release to TestPyPI from workflow dispatch. If this isn't of interest this can get removed.

Reply:
This is worth doing at least the first time around. How do you do it? Is there some option when you create the release in the GitHub interface?

Reply:
You do this through workflow dispatch by visiting the GitHub Actions tab for the repo and then selecting this workflow (https://github.com/data-apis/array-api-compat/actions/workflows/publish-package.yml) and then when you click the "Run workflow" button on the right hand side there will be the option to select `false` or `true` from a dropdown.

This is how it looks like for us on `pyhf`'s GitHub repo:

![pyhf-view](https://github.com/data-apis/array-api-compat/assets/5142394/64b0549c-d506-45a5-bae1-7b7da42d69b4)

and if it helps, starting around 52 seconds into this video of me making the `pyhf` `v0.7.1` release I show this button example: https://youtu.be/ZV20tr3EpTw?t=52